hmpps-prisoner-search-indexer/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/resource/DomainEventsResource.kt

```diff
@@ -51,6 +51,73 @@ class DomainEventsResource(private val domainEventEmitter: HmppsDomainEventEmitt
       occurredAt = occurredAtTime,
     )
   }
+
+  @PutMapping("/events/prisoner/released/{prisonerNumber}")
+  @ResponseStatus(HttpStatus.ACCEPTED)
```

Comment:
should this be `ACCEPTED`?  Would `NO_CONTENT` be more appropriate since the event is sent immediately?